src/objc_glue.c

```diff
@@ -145,7 +145,9 @@ Symbol *objc_getMethVarName(const char *s, size_t len)
 
 Symbol *objc_getMethVarName(Identifier *ident)
 {
-    return objc_getMethVarName(ident->string, ident->len);
+    const char *id = ident->toChars();
```

Comment:
Is this the correct fix? Isn't it better to make the fields in `Identifier` public?

Reply:
At one point, this branch and 2.076 branch were sharing the same C++ headers (minus ctors/dtors), this was important to make sure that I could switch between the C++ and D front-ends without requiring any changes to the glue.

I'd rather just leave it as is and just fix up the places that aren't part of the common front-end.